<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateStudent</class>
 <widget class="QDialog" name="UpdateStudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="saveBtn">
        <property name="text">
         <string>Sauvegarder</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel</set>
        </property>
        <property name="centerButtons">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>446</width>
        <height>878</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Mise a jour des informations&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Nom*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lastname">
           <property name="placeholderText">
            <string>Entrer le nom</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Prenom(s)*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="firstname">
           <property name="placeholderText">
            <string>Entrer le(s) prenom(s)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_18">
         <item>
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Situation</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="situation">
           <item>
            <property name="text">
             <string>invited</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>admited</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>unpaid</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Email*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="email">
           <property name="placeholderText">
            <string>john@exemple.com</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Niveau*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="level">
           <item>
            <property name="text">
             <string>L1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>L2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>L3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>M1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>M2</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>CIN*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="cin">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Entrer le numero cin</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_14">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Telephone*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="phone">
           <property name="placeholderText">
            <string>*** ** *** **</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="7" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_17">
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Parcours*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="category">
           <item>
            <property name="text">
             <string>Pro</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Gb</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sr</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ig</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="8" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_16">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Bac*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="bac">
           <item>
            <property name="text">
             <string>C</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>D</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>S</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>T</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="9" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Delivrance de la CIN*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="deliveranceDate"/>
         </item>
        </layout>
       </item>
       <item row="10" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Duplicata de la CIN</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="duplicata">
           <property name="placeholderText">
            <string>Entrer le numero du duplicata</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="11" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Delivrance du duplicata</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="duplicataDate"/>
         </item>
        </layout>
       </item>
       <item row="12" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Annee d'inscription*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="inscriptionYear">
           <property name="minimum">
            <number>2022</number>
           </property>
           <property name="maximum">
            <number>2100</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="13" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Annee de naissance*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="birthday"/>
         </item>
        </layout>
       </item>
       <item row="14" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Lieu de naissance*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="birthplace"/>
         </item>
        </layout>
       </item>
       <item row="15" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_13">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Nom de la mere*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="motherName"/>
         </item>
        </layout>
       </item>
       <item row="16" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_15">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Nom du pere*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="fatherName"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UpdateStudent</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UpdateStudent</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
